tgui/packages/tgui/interfaces/NtosEmojipedia.js

```diff
@@ -0,0 +1,76 @@
+import { useBackend, useSharedState } from '../backend';
+import { Box, Button, Input, Section } from '../components';
+import { NtosWindow } from '../layouts';
+import '../styles/interfaces/Emojipedia.scss';
+
+export const NtosEmojipedia = (props, context) => {
+  const { data } = useBackend(context);
+  const { emoji_list } = data;
+  const [filter, updatefilter] = useSharedState(context, 'filter', '');
+
+  let filtered_emoji_list = filter
+    ? emoji_list.filter((emoji) => {
+      return emoji.name.toLowerCase().includes(filter.toLowerCase());
+    })
+    : emoji_list;
+  if (filtered_emoji_list.length === 0) {
+    filtered_emoji_list = emoji_list;
+  }
+
+  return (
+    <NtosWindow width={600} height={800}>
+      <NtosWindow.Content scrollable>
+        <Section
+          title={'Emojipedia V2.2.1-pre' + (filter ? ` - ${filter}` : '')}
+          buttons={
+            <>
+              <Input
+                type="text"
+                placeholder="Search by name"
+                value={filter}
+                onInput={(_, value) => updatefilter(value)}
+              />
+              <Button
+                tooltip={'Click on an emoji to copy its tag!'}
+                tooltipPosition="bottom"
+                icon="circle-question"
+              />
+            </>
+          }
+          display="grid"
+          box-sizing="border-box"
+          margin="0"
+          padding="0"
+          grid-template-columns="repeat(auto-fill, 32px)"
+          grid-template-rows="repeat(auto-fill, 32px)"
+          grid-gap="4em">
```

Comment:
These properties, starting with display, aren't supposed to be working on Box and should better stay in CSS, and CSS grid support in IE11 is a little bad, keep that in mind, as long as it works as you expect.